### PR TITLE
cgame: declare ui_winner locally, it's read by the HUD on every frame

### PR DESCRIPTION
**Pull request by illwieckz**
Declare `ui_winner` locally, it's read by the HUD on every frame.

### CODE REVIEW

File: src/cgame/cg_main.cpp [hunk 1]
```diff
@@ -174,6 +174,7 @@ static Cvar::Cvar<float> cg_fov_level4("cg_fov_level4", "field of view (degrees)
 static Cvar::Cvar<float> cg_fov_human("cg_fov_human", "field of view (degrees) for humans", Cvar::NONE, 0);
 
 Cvar::Cvar<bool> ui_chatPromptColors("ui_chatPromptColors", "chat prompts (e.g. 'Say:') are color-coded", Cvar::NONE, true);
+Cvar::Cvar<std::string> ui_winner("ui_winner", "FOR INTERNAL USE", Cvar::NONE, "");
```
Comment (DolceTriade) [inline src/cgame/cg_main.cpp:177]: Cvar::ROM and give proper description.

Comment (illwieckz) [reply src/cgame/cg_main.cpp:177]: We already use `FOR INTERNAL USE` with `cg_minimapActive`, the fact those are cvars is basically a HACK and we better implement a specific features for variables internal to the cgame and the ui.

Comment (illwieckz) [reply src/cgame/cg_main.cpp:177]: If this cvar is `Cvar::ROM` then `ui_winner` can never be updated and then when the game ends the cvar remains empty and the scoreboard doesn't display the winning team name.

Comment (DolceTriade) [reply src/cgame/cg_main.cpp:177]: Are you sure? Cvar::ROM just means that users cannot modify it. The value can be changed with `Cvar::SetValue()`

Comment (slipher) [reply src/cgame/cg_main.cpp:177]: ROM, like LATCH, is not yet implemented for the gamelogic.

Comment (slipher) [reply src/cgame/cg_main.cpp:177]: illwieckz is right that this should not be a cvar anyway. We could make something like a data source but for singletons

**Comment by slipher** [comment]
LGTM

**Review by DolceTriade — APPROVED** [review]

**Pull request merged**